packages/design-system/src/components/Checkbox/Checkbox.tsx

```diff
@@ -52,7 +52,7 @@ const CheckboxEl = React.forwardRef<CheckboxElement, CheckboxElProps>(
 
     return (
       <CheckboxRoot ref={composedRefs} checked={checked} onCheckedChange={setChecked} {...props}>
-        <CheckboxIndicator style={{ display: 'inline-flex', pointerEvents: 'auto' }} forceMount>
+        <CheckboxIndicator style={{ display: 'inline-flex', pointerEvents: 'auto'}} forceMount>
```

Comment:
Sorry maybe my previous comment wasn't clear. Could we add these to the styled component as well so all styles are applied in the same place and we can get rid of the style tag here?

Reply:
I see now, i agree it is redundant. Ill make that change.

Thanks for clarifying!

Reply:
Great thanks!